internal/provider/provider_sdk2.go

```diff
@@ -108,8 +109,8 @@ func getDataSourcesMap() map[string]*schema.Resource {
 		"mongodbatlas_api_keys":                          mongodbatlas.DataSourceAPIKeys(),
 		"mongodbatlas_access_list_api_key":               mongodbatlas.DataSourceAccessListAPIKey(),
 		"mongodbatlas_access_list_api_keys":              mongodbatlas.DataSourceAccessListAPIKeys(),
-		"mongodbatlas_project_api_key":                   mongodbatlas.DataSourceProjectAPIKey(),
-		"mongodbatlas_project_api_keys":                  mongodbatlas.DataSourceProjectAPIKeys(),
+		"mongodbatlas_project_api_key":                   projectapikey.DataSourceProjectAPIKey(),
```

Comment:
[nit-non-blocking] given the decision on one package per resource and to follow the "don't stutter" general recommendation of go 
```suggestion
		"mongodbatlas_project_api_key":                   projectapikey.DataSource(),
```

Reply:
this makes sense, will do a followup PR to adjust all of the resources that have already been adjusted to new structure.

Reply:
nice, thx